src/util-device.h

```diff
@@ -20,27 +20,31 @@
 
 #include "queue.h"
 #include "unix-manager.h"
+#include "runmodes.h"
 
 /** storage for live device names */
 typedef struct LiveDevice_ {
     char *dev;  /**< the device (e.g. "eth0") */
+    enum RunModes runmode; /**< the runmode (e.g. "RUNMODE_NFLOG") */
     int ignore_checksum;
+    void (*offload_callback)(Packet *p);
```

Comment:
Our naming for functions and function pointers means this should be OffloadCallback